conf/modules/prepare_genome.config

```diff
@@ -76,7 +76,7 @@ process {
     }
 
     withName: 'TABIX_DBSNP' {
-        ext.when         = { !params.dbsnp_tbi && params.dbsnp && ((params.step == "mapping" || params.step == "markduplicates" || params.step == "prepare_recalibration") || params.tools && (params.tools.split(',').contains('controlfreec') || params.tools.split(',').contains('haplotypecaller') || params.tools.split(',').contains('mutect2'))) }
+        ext.when         = { !params.dbsnp_tbi && params.dbsnp && (params.step == "mapping" || params.step == "markduplicates" || params.step == "prepare_recalibration") || params.tools && ( params.tools.split(',').contains('controlfreec') || params.tools.split(',').contains('haplotypecaller') || params.tools.split(',').contains('mutect2')) }
```

Comment:
why are you removing the `(` here? It changes the expression I think:


0 && 1 when dbsnp doesn't exists and dbsnp does
AND ( (when starting from mapping/md/prepare_recal)  OR Controlfreec OR Haplytypecalle OR Mutect )

!= AND (when starting from mapping/md/prepare_recal)  OR Controlfreec OR Haplytypecalle OR Mutect

Reply:
Good catch, I think I deleted them by mistake when I removed the now obsolete `_MS` subwfs.